File: CyberHunters/Assets/Scripts/spectrum/checkpoint.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class checkpoint : MonoBehaviour {

    public GameObject destination;

	void OnTriggerEnter2D(Collider2D col)
    {
        if (col.CompareTag("Player"))
        {
            col.transform.position = destination.transform.position;
        }
    }
}

File: Java/Assets/_SPECTRUM/scripts/checkpoint.java
import unityengine.*;

public class checkpoint extends MonoBehaviour
{

	public GameObject destination;

	private void OnTriggerEnter2D(Collider2D col)
	{

		if (col.CompareTag("Player"))
		{
			col.transform.position = destination.transform.position;
			Handheld.Vibrate();
		}
	}

}